<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NetAssistant</class>
 <widget class="QWidget" name="NetAssistant">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>938</width>
    <height>803</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>NetAssistant</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,1">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>网络设置</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0">
          <item>
           <layout class="QFormLayout" name="formLayout">
            <item row="0" column="0">
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>设置模式：</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QComboBox" name="setModeBox">
              <item>
               <property name="text">
                <string>TCP客户端</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>TCP服务端</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>UDP</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="target_ip_label">
              <property name="text">
               <string>服务器ip:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="target_ip">
              <property name="inputMask">
               <string>000.000.000.000</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="target_port_label">
              <property name="text">
               <string>服务器端口：</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLineEdit" name="target_port">
              <property name="inputMask">
               <string>99999</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="cursorPosition">
               <number>0</number>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
              <property name="placeholderText">
               <string>0-65535</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="connectBtn">
            <property name="text">
             <string>连接</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,0,0">
       <item>
        <widget class="QTextEdit" name="msgTextEdit"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,0,0,1,0,0">
         <property name="leftMargin">
          <number>10</number>
         </property>
         <property name="rightMargin">
          <number>10</number>
         </property>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>编码：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="encodeBox">
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>16777215</height>
            </size>
           </property>
           <item>
            <property name="text">
             <string>UTF-8</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>GBK</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>本地IP:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="local_ip"/>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>本地端口：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="local_port">
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="rightMargin">
          <number>10</number>
         </property>
         <item>
          <widget class="QTextEdit" name="sendTextEdit"/>
         </item>
         <item>
          <widget class="QPushButton" name="sendMsgBtn">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>发送</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
